script.js

```diff
@@ -1,12 +1,200 @@
 //You can edit ALL of the code here
+
+let allEpisodes;
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  root.innerText = '';
+
+  let showContainer = document.getElementById("show-container");
+  if (showContainer) {
+    showContainer.innerText = '';
+  } else {
+    showContainer = document.createElement("div");
+    showContainer.id = "show-container";
+    root.appendChild(showContainer);
+  }
+
+  const searchBox = document.createElement("input");
```

Comment:
We'll discuss it for real elsewhere, but here I might comment that the generation of these elements is not really dependent on any data, so is perhaps best done in... HTML.  
Blah blah visibility and moving the section between parents (if necessary)